libs/client-sdk/src/bindings/c/sdk.cpp

```diff
@@ -291,5 +291,37 @@ LD_EXPORT(void) LDClientSDK_Free(LDClientSDK sdk) {
     delete TO_SDK(sdk);
 }
 
+LD_EXPORT(LDListenerConnection)
+LDClientSDK_FlagNotifier_OnFlagChange(LDClientSDK sdk,
+                                      char const* flag_key,
+                                      struct LDFlagListener listener) {
+    LD_ASSERT_NOT_NULL(sdk);
+    LD_ASSERT_NOT_NULL(flag_key);
+
+    auto connection = TO_SDK(sdk)->FlagNotifier().OnFlagChange(
```

Comment:
If `listener.FlagChanged` is null, then is there any need to register a callback?

Reply:
I guess we could re-word the documentation to indicate that the returned connection can be NULL if no listener is registered.

Currently the need to register a listener is just to get the connection.